content/docs-v1/citation-bibliographies.md

```diff
@@ -33,72 +33,110 @@ entries:
+```
+{{< /q-class >}}
+
+## Display a Bibliography
+
+In Quire projects you can include both page-level bibliographies and a complete bibliography for the entire publication.
 
-The `q-cite` shortcode can be used anywhere in your Markdown text, including within footnotes.
+Both bibliography styles can appear in either a full or short format. This is controlled globally by the `displayShort` attribute found in the `content/_data/config.yaml`.  The default is set to `true`. If you want to list the references in full, change the value to `false`.
 
-If a publication includes in-text citations and appears to have missing bibliographic entries, there is most likely an incorrect or missing `q-cite` shortcode in the essay end notes. You can find those in the essay Markdown file. The quoted text that follows `q-cite` should exactly match the “id” in the `references.yaml` files. Any changes made to a bibliographic citation should be made first to the `references.yaml` file, and then to every instance of that citation throughout the text. Failure to make changes in all places will result in broken links. The `q-cite` shortcode also plays an important role when generating a bibliography. So a missing or misspelled `q-cite` shortcode will result in the entry entry being left out of the bibliography completely.
+```yaml
+  bibliography:
+    displayShort: true
+```
 
+### Page-level Bibliography
 
+If you are using in-text citations your publication will automatically include a page-level bibliography listing all works that were cited on that page using the `{% cite %}` shortcode.
 
-## Display a Bibliography
+If a publication appears to have missing bibliographic entries there is most likely an incorrect or missing `{% cite %}` shortcode in the essay end notes. The `id` used in the `{% cite %}` shortcode must exactly match the `id` in the `references.yaml` files.
+
+If you want to suppress the page level bibliography but keep the in-text citations, go into the `content/_data/config.yaml` and change the value to `false`:
+
+```yaml
+  bibliography:
+    displayOnPage: false
+```
 
-Your publication will automatically include a page-level bibliography listing all works that were cited on that page using the `q-cite` shortcode. A heading can be customized to go above the page bibliography with the `biblioHeading` parameter in your `config.yaml` file. The default is "Bibliography".
+If you do not want to use in-text citations but still want to include a page-level bibiliography you have a two options:
 
-Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you can create a `bibliography.md` file and use the `q-bibliography` shortcode. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file.
+- Add a page-level biblio using Markdown
```

Comment:
Though I suggested this as an option in that GitHub issue, I don't think we should necessarily include it here. Better to assume users will work with refereences.yaml and not float the idea of just dumping it into markdown. Or, if you do want to keep it, I'd make it the second option, but then elaborate just a little more ....

> Skip the use of references.yaml, and instead add your bibliography directly into the page with Markdown, using list or definition list formatting

Markdown lists we have documented but we might not have definition lists included? But they work, here's a sample: 

```
apple
: a fruit you give teachers

banana
: mushy to eat, funny to slip on

orange
: a fruit and a color
```